app/javascript/react/locales/en/translation.json

```diff
@@ -49,7 +49,7 @@
     "localTime": "{{lastUpdate}} (local time)",
     "legendTitle": "Measurements legend",
     "calendarTitle": "Measurements calendar",
-    "measurementsUnits": "(in µg/m)"
+    "measurementsUnits": "(in µg/m3)"
```

Comment:
not listed in trello task, but done intentionally right?

Reply:
Yes :)